<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>547</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1073</width>
    <height>547</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1073</width>
    <height>547</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="query_tab">
       <attribute name="title">
        <string>查询</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>50</y>
          <width>461</width>
          <height>42</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4,1">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>18</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>卡信息:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="card_info_line">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="query_btn">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>搜索</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>50</y>
          <width>441</width>
          <height>42</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,1">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>18</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>订单号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="order_no_box">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentText">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_btn">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="result_group_box">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>1001</width>
          <height>271</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,7">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>18</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>文件名：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="filename_line">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,7">
           <item>
            <widget class="QLabel" name="iccid_label">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>18</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>ICCID:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="iccid_line">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,7">
           <item>
            <widget class="QLabel" name="puk1_label">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>18</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>PUK1:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="puk1_line">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="not_found_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>230</y>
          <width>301</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>20</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;未查询到卡片来源&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="query_label">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>231</y>
          <width>201</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>正在查询 ...</string>
        </property>
       </widget>
       <widget class="QLabel" name="query_gif_label">
        <property name="geometry">
         <rect>
          <x>382</x>
          <y>240</y>
          <width>32</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Tex</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="config_tab">
       <attribute name="title">
        <string>设置</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>11</y>
          <width>1011</width>
          <height>211</height>
         </rect>
        </property>
        <property name="title">
         <string>数据库</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,5,1,5,1">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>主机：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="host_line">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>端口：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="port_line">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="1,5,1,5,1">
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>用户名：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="username_line">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>密码：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="password_line">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="1,5,1,5,1">
           <item>
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,3">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>订单数据库：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="data_database_line">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>打印数据库：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="print_database_line">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_12">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>240</y>
          <width>331</width>
          <height>181</height>
         </rect>
        </property>
        <property name="title">
         <string>系统</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>120</y>
           <width>231</width>
           <height>37</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="5,1">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>线程池个数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="thread_count_spin_box">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>70</y>
           <width>231</width>
           <height>37</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="5,1">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>连接池个数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="connect_count_spin_box">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QPushButton" name="save_btn">
        <property name="geometry">
         <rect>
          <x>920</x>
          <y>380</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
